docs/setup-guide/scale-out-poc-setup.md

```diff
@@ -1,40 +1,27 @@
-### Setup proxy
+### Brief 
 
-Please follow [these steps](https://github.com/futurewei-cloud/arktos-perftest/wiki/How-to-setup-and-config-nginx-proxy-for-scale-out-design#setting-up-proxy). 
-
-The proxy can run on a machine of your choice. Make sure to update the ip addresses in the Nginx config for your partition servers.
-
-This is a one-time setup and pretty straightforward so it's not included in this PR as a script. 
-
-Will automate this in another PR.
-
-### Set proxy endpoint:
-
-This to to be done on ***both tenant and resource partitions***.
-
-Set the proxy url like
-
-```bash
-export SCALE_OUT_PROXY_ENDPOINT="http://192.168.0.120:8888"
-```
+This introduces the tool to set up the tenant-partition cluster and the resource-partition cluster.
 
 ### On the TENANT partition:
 
 Run 
 ```bash
-IS_RESOURCE_PARTITION=false arktos-up-scale-out-poc.sh
+IS_RESOURCE_PARTITION=false INSTALL_SCALE_OUT_PROXY=true SECOND_PARTITION_IP=[resource-partition-ip] hack/arktos-up-scale-out-poc.sh
 ```
 
-This will start all the components except kubelet, kubeproxy, and nodelifecycle and nodeipam controllers are disabled
+This will 
+1. install the nginx proxy in the local host of tenant-partition, 
+2. start all the components except kubelet, kubeproxy,
+3. disable nodelifecycle and nodeipam controllers.
 
 ### On the RESOURCE partition:
 
 Run 
 ```bash
-IS_RESOURCE_PARTITION=true arktos-up-scale-out-poc.sh
+IS_RESOURCE_PARTITION=true SCALE_OUT_PROXY_ENDPOINT=http://[tenant-partition-ip]:8888 hack/arktos-up-scale-out-poc.sh
 ```
 
 This will start all the components except scheduler, and only nodelifecycle and nodeipam controllers are enabled
 
 ### Tests done
```

Comment:
this section is probably not needed. it was copied from the PR description

Reply:
removed